packages/react/src/components/form-photo-uploader/FormPhotoUploader.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useCallback, useState } from 'react';
+
+import './FormPhotoUploader.scss';
+
+import { FileUploader } from 'devextreme-react/file-uploader';
+
+export const FormPhotoUploader = () => {
+  const [isDropZoneActive, setDropZoneActive] = useState(false);
+
+  const onDropZoneEnter = useCallback((e) => {
+    if (e.dropZoneElement.id === 'uploader') {
+      setDropZoneActive(true);
+    }
+  }, [isDropZoneActive]);
```

Comment:
These two hooks will change every time isDropZoneActive changes, it's unnecessary in this case. Pass the empty array instead.

Reply:
Fix